frontend/lib/norent/data/faqs-content.tsx

```diff
@@ -8,13 +10,29 @@ export type FaqCategory =
   | "After Sending Your Letter"
   | "States with Limited Protections";
 
+type FaqCategoryLabels = {
+  [k in FaqCategory]: string;
+};
+
+export function getLabelNameForFaqCategory(): FaqCategoryLabels {
```

Comment:
Since this is actually returning an object that represents all labels, maybe this should be called `getLabelsForFaqCategories`?  Or if you want to have it be in keeping w/ the function names for choices in the common data directory, `getFaqCategoryLabels()`?

Reply:
Sure thing